pinecone/index_connection_test.go

```diff
@@ -256,6 +256,13 @@ func (ts *IntegrationTests) TestUpdateVectorValues() {
 	assert.ElementsMatch(ts.T(), expectedVals, actualVals, "Values do not match")
 }
 
+func (ts *IntegrationTests) TestUpdateVectorMissingReqdFields() {
```

Comment:
Same as above, I think this can be a unit tests since we're expecting a client error before making the request.